tests/tcp/test_counter_tcp_ports_e2e.py

```diff
@@ -57,35 +53,22 @@ def captures_ok(api, cfg, size, utils):
     """
     src = [
```

Comment:
why we reduced this coverage. I feel we do not have any model change in config path.

Reply:
Temporarily reduced coverage 67 -> 0 for initial CI. reverted to 67